ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -89,42 +90,38 @@ bool SysAlarmsMessagingPlugin::initialize(const QStringList &arguments, QString
  */
 void SysAlarmsMessagingPlugin::updateAlarms(UAVObject* systemAlarm)
 {
-    Q_ASSERT(systemAlarm->getObjID() == SystemAlarms::OBJID);
```

Comment:
So this check should remain since the code below references constant definitions from the generator. 

Basically any time this error would fire, now you can get unpredictable/undefined behavior.

You can do this safely but it needs to be consistently a runtime constant, so do a string value comparison.